res/database/Default/database_news.xml

```diff
@@ -1439,10 +1679,12 @@ Her normal body size is expected to be reached in a week.</en>
 			<title>
 				<de>Welt lacht über ${animal_${index}}-%minister%</de>
 				<en>World laughs at ${animal_${index}} %minister%</en>
+				<pl>Świat śmieje się z urzędnika i jego rozporządzenia.</pl>
```

Comment:
no variable used on purpose?

Reply:
I've changed this. %minister% in front of sentence, because Capital letter.